tests/beta/test_default_tensor_tf.py

```diff
@@ -632,7 +633,7 @@ def cost(self, interface, torch_support):
         if interface == "torch" and not torch_support:
             pytest.skip("Skipped, no torch support")
 
-        @qnode(dev, diff_method="best", interface=interface)
+        @qnode(dev, diff_method="device", interface=interface)
```

Comment:
It seems that there's still a validation check that will have to be disabled with the `PassThruQNode`, when using decompositions. Giving this a go with `"classical"`:
```python
~/pennylane/pennylane/operation.py in __init__(self, wires, do_queue, *params)
    328         if self.do_check_domain:
    329             for p in params:
--> 330                 self.check_domain(p)
    331         self.params = list(params)  #: list[Any]: parameters of the operator
    332 

~/pennylane/pennylane/operation.py in check_domain(self, p, flattened)
    425             if not isinstance(p, numbers.Real):
    426                 raise TypeError(
--> 427                     "{}: Real scalar parameter expected, got {}.".format(self.name, type(p))
    428                 )
    429 

TypeError: Rot: Real scalar parameter expected, got <class 'tensorflow.python.framework.ops.EagerTensor'>.
```

Whereas changing the direct call on `U3` to its decomposition (` qml.U3.decomposition(x, weights[0], weights[1], wires=w)`) seemed to have helped out.